index.js

```diff
@@ -96,7 +96,13 @@ function formatNode(node) {
     if (node.children && node.children.length) {
         log += dim('[') + yellow(node.children.length) + dim(']');
     } else {
-        log += dim(': \'') + green(node.value) + dim('\'');
+        log += dim(': ') + green(JSON.stringify(node.value));
```

Comment:
var paragraph = 'Some simple text.\nOther $B!H(Bsentence$B!I(B.';

In this case,
I want to show '\n' value.

Easy way is to use JSON.stringfy() .

I like single quote too.
but I could not output '\n' instead of  "\n".

2015/10/03 7:34$B!"(BTitus Wormer <notifications@github.com> $B$N%a%C%;!<%8(B:

> In index.js:
> 
> > @@ -96,7 +96,13 @@ function formatNode(node) {
> >      if (node.children && node.children.length) {
> >          log += dim('[') + yellow(node.children.length) + dim(']');
> >      } else {
> > -        log += dim(': \'') + green(node.value) + dim('\'');
> > +        log += dim(': ') + green(JSON.stringify(node.value));
> Oh just noticed that it has to do with \n and \r!
> 
> $B!=(B
> Reply to this email directly or view it on GitHub.
>